### PR TITLE
Fixes broken link in What is Lisk Core page in Core docs v3-beta-5 - Closes #1057

### LINKED ISSUE #1057: Fix broken link in What is Lisk Core page in Core docs v3 beta-5
Fix broken link in 'What is Lisk Core?' page.

### PR DESCRIPTION
**Pull request by chris529**
Closes #1057

### CODE REVIEW

**Review by Tschakki — APPROVED** [review]

**Review by Tschakki — CHANGES_REQUESTED** [review]
@chris529 I just noticed there is a conflict in this PR, please resolve it, so I can proceed with the merge

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -22,7 +22,9 @@ Mona Bärenfänger <mona@lightcurve.io>
 :url_semver: https://semver.org/
 :url_swagger: https://swagger.io
 // Project URLs
-:url_config: management/index.adoc
+:url_admin_binary_snapshot: management/application.adoc#create_snapshot
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: I think this link can be removed.

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -22,7 +22,9 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+:url_config: management/configuration.adoc
+:url_config_api: {v_sdk}@lisk-sdk::guides/node-management/api-access.adoc
```
Comment (Tschakki) [inline modules/ROOT/pages/index.adoc]: This link can bbee removed as well, it is not used on the page.

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: 👍

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/index.adoc]: 👍

**Review by Tschakki — APPROVED** [review]

**Pull request merged**